sections/event-types.txt

```diff
@@ -2023,6 +2023,21 @@ myDiv.addEventListener("auxclick", function(e) {
 	deltaY value respectively).
 	</p>
 
+	A <a>user agent</a> SHOULD create a <a>wheel event group</a> when the first wheel event
```

Comment:
I would make this "MUST" so that it's well-defined what the `target` is for `wheel` events. The UA-defined timeout allows UAs to not implement the grouping already.